src/main/features/first-contact/routes/claim-reference.ts

```diff
@@ -25,13 +29,17 @@ export default express.Router()
       const form: Form<ClaimReference> = req.body
 
       if (form.hasErrors()) {
+        if (isCCBCCaseReference(form.model.reference)) {
```

Comment:
Out of curiosity why didn't we just check whether reference matches CMC format, it would not require putting list of CCBC identifiers in the code.

Reply:
business request... I asked the same thing